### PR TITLE
Sc 243 initial db config interfaces and models

### PR DESCRIPTION
**Pull request by marialobillo**

### CODE REVIEW

**Comment by AbdelrahmanRadwan** [comment]
For each commit, try to add a summary of what you added, so the reviewer would have some context.

File: server/sample.env
Comment (AbdelrahmanRadwan) [inline server/sample.env:1]: If by this you mean to share sample of how the env should look like, then I think this should be part of the README instead.

File: server/src/models/Service.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    staffId: { type: String, required: true },
+    name: { type: String, required: true },
+    duration: { type: Number, required: true },
+    price: { type: Number, required: true },
+  },
+  {
+    timestamps: true,
+    versionKey: false,
+  }
+);
```
Comment (AbdelrahmanRadwan) [inline server/src/models/Service.ts:15]: I don't understand what is this, and what is it for?

**Review by AbdelrahmanRadwan — CHANGES_REQUESTED** [review]

File: server/src/models/Client.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  {
+    timestamps: true,
+    versionKey: false,
+  }
+);
+
+const ClientModel = model<IClient>('Client', ClientSchema);
+
+export default ClientModel;
```
Comment (AbdelrahmanRadwan) [inline server/src/models/Client.ts]: What is the different between a client and a staff?

File: server/build/config/connectionDB.js
Comment (AbdelrahmanRadwan) [inline server/build/config/connectionDB.js:1]: The files under build shouldn't be pushed. Build should be added to gitignore.

File: server/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+const app = express()
+
+// Middlewares
+app.use(cors())
+app.use(helmet())
+
+
+app.get('/', (req: Request, res: Response) => {
+	res.send
+('Hiii!')
```
Comment (AbdelrahmanRadwan) [inline server/src/index.ts]: for testing?

**Review by AbdelrahmanRadwan — CHANGES_REQUESTED** [review]

File: server/sample.env
Comment (marialobillo) [reply server/sample.env:1]: thank you, I think it's done, and I tried to add better summary on the commit message

File: server/src/models/Service.ts (hunk 1 above)
Comment (marialobillo) [reply server/src/models/Service.ts:15]: I thought I need a schema for create the model for every collection on mongo, like here. Am I wrong?

const ClientModel = model<IClient>('Client', ClientSchema);

File: server/src/models/Client.ts (hunk 2 above)
Comment (marialobillo) [reply server/src/models/Client.ts]: You are right, I am not sure if I want to add more attributes to the Staff or to the Client collection, so I decided to put it separately, but I could do just User model.

File: server/src/index.ts (hunk 3 above)
Comment (marialobillo) [reply server/src/index.ts]: Yes, it was for testing, I did remove it

File: server/src/models/Client.ts (hunk 2 above)
Comment (marialobillo) [reply server/src/models/Client.ts]: I have removed client and staff, and I added user and role instead.

File: server/build/config/connectionDB.js
Comment (marialobillo) [reply server/build/config/connectionDB.js:1]: Thanks, I added build folder to the gitignore file

**Pull request closed without merging**